docs/bootstrap.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ID": "bootstrap",
+  "Properties": {
+    "Resources": {
+      "resource_a": {
+        "Plugin": "resource-file",
+        "Type": "a",
+        "Properties": {
+          "Note": "Depends on {{.Resources.resource_b.ID}}"
```

Comment:
I'd figured references would always occur inside strings, largely because in working with a prototype AWS resource plugin, IDs turned out to be the only useful dynamic value. That may well be an artifact of my use case, but it's worth thinking through now because IDs are the only dynamic value currently returned by Provision and DescribeInstances/DescribeResources.